sdccc/src/main/java/com/draeger/medical/sdccc/TestSuite.java

```diff
@@ -594,17 +594,21 @@ private static void printVerdict(final long numberOfTestFailures, final File tes
         }
 
         if (numberOfTestFailures == 0) {
-            LOG.info(
-                    "Test run with {} Tests was completed successfully. No problems were found.",
-                    testRunObserver.getTotalNumberOfTestsRun());
+            if (testRunObserver.isInvalid()) {
+                LOG.info(
+                        "Test run with {} Tests was completed successfully but other problems were found. Please consult the logfiles in {} for further information.",
+                        testRunObserver.getTotalNumberOfTestsRun(),
+                        testRunDir);
+                LOG.info("Test run was invalid.");
+            } else {
+                LOG.info(
+                        "Test run with {} Tests was completed successfully. No problems were found.",
+                        testRunObserver.getTotalNumberOfTestsRun());
+                LOG.info("Test run was valid.");
+            }
         } else {
             LOG.info("Test run found problems. Please consult the logfiles in {} for further information.", testRunDir);
-        }
-
-        if (testRunObserver.isInvalid()) {
             LOG.info("Test run was invalid.");
```

Comment:
Remove this line, after moving line 597 out of the outer conditional block.

Reply:
done